Program.cs

```diff
@@ -5,11 +5,91 @@ namespace Calculator
   public static class Program
   {
     private static void Main(string[] args)
+    {
+      PrintWelcomeMessage();
+
+      while (true)
+      {
+        PerformOneCalculation();
+      }
+    }
+
+    private static void PrintWelcomeMessage()
     {
       Console.WriteLine("Welcome to the calculator!");
       Console.WriteLine("==========================");
+    }
+
+    private static void PerformOneCalculation()
+    {
+      var op = AskForOperator();
+      var n = AskForNumberArray(op);
+      var a = CalculateAnswer(op, n);
+
+      Console.WriteLine("The answer is: " + a);
+      Console.WriteLine();
+    }
+
+    private static string AskForOperator()
+    {
+      string[] allowedOperators = { "+", "-", "*", "/" };
```

Comment:
Making the string allowedOperators readonly means that it cannot be changed later during runtime. Also, currently, this variable is not used so user input is not validated, a separate method that checks this would be good i.e.
private static bool IsValidOperator(string input)
{
      return allowedOperators.Contains(input);
}